interview.md

```diff
@@ -11,41 +11,61 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+This idea exites me and other poker players because it combines this notion of a ststitical game with more accurate stastitics as well as provides the player the ability to keep track of past hands without having to recall memory from the last 4 hours of play. I was able to come up with this idea after interveiwing my dad who is a sofware engineer and a poker player. After having a long converstion about things he does at work and things that he does in his free time we narrowed in on poker. Since we both play poker we both found it difficult to keep track of things that happen at the poker table espessally since you can be so focused on the game. Additionally good pker player always write down intersting events that happen at the table such as big hands, how they busted out, and tells of oher players. This idea of a dsl that allows for real time tracking of these events came to life. 
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+real time Language for poker analysis
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+The user would be able to use natural laguage to describe poker hands and events or even small key words/ operations to decribe events at the table. Such as "in my hand I held As(Ace of spades) 2s(2 of spades) I raise from 10$ big blid to 30$ preflop There is one caller, the flop comes ..., the turn comes ..., the oponent in ___ position bet ..., I called ..., the river card was a ..."
+Another possible way would to use short function calls such as:
+newhand(), myHand(), opponet(position on table, what was their action), Ibet(), EndHand() ect... a bunch of poker related functions that store quick and easy info about the table. 
+The goal would be to use a language that allows for quick execution by the user, so that their attention does not devide from the table for more then a few moments. 
+
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program would save user poker infromation within a data sructure or database when the user makes a command. The types of errors that would be possible could be from the use of natural language processing or a bug in the sysytem with storage. These errors can be comunicted through direct feedback upon a user command. Additionally since poker is a game of percentages feedback on play can be updated by providing statistics back to the player upon each command given to the system. 
+
```

Comment:
I like the idea of constant feedback - so the user can know without thinking too much how that new data maybe changed something.